set.mm

```diff
@@ -426213,6 +426213,111 @@ Norman Megill (2007) section 1.1.3.  Megill then states, "A number of
 
 $(
 =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+                          Aristotelian logic
+
+Model Aristotelian logical syllogisms using modern notation.
+This shows that that its syllogisms can be proven with our axioms of logic.
+In antiquity Aristotelian logic and Stoic logic
+(see ~ mpto1 ) were the leading logical systems.
+Aristotelian logic became the leading system in medieval Europe;
+the medieval schools formed form mnemonic names such as "Barbara"
+to name the different syllogisms.
+
+Here we model the syllogisms using wff variables
+to make them more broadly applicable.
+As a result, these syllogisms are somewhat more general
+than traditional Aristotelian logic (where the predicates are
+only allowed to have certain restricted forms).
+
+Aristolean logic is essentially the forerunner of predicate calculus
+(with a hint of set theory since it discusses membership in groups),
+while Stoic logic is essentially the forerunner of propositional calculus.
+
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
```

Comment:
The section comment goes after the bar, not before, unless you want this all to be part of the section title. (Reminder that you can push another commit to the branch to fix this instead of closing the PR.)